lab_1/main.py

```diff
@@ -12,9 +24,23 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
+    tokens = []
+    if type(text) == str:
+        text = text.lower()
+        for symbol in text:
+            if symbol in '''1234567890-=!@#$%^&*()_+`~[]\{}|;':",./<>?''':
+                text = text.replace(symbol, '')
+        text = text.split()
+        tokens.extend(text)
```

Comment:
No need for `extend`, you can assign `text.split()` to the `tokens`.

Reply:
this was done to return an empty list. I thought writing 'return []' was too easy